ts/src/base/Exchange.ts

```diff
@@ -1460,8 +1461,14 @@ export default class Exchange {
     }
 
     async close () {
+        // test by running ts/src/pro/test/base/test.close.ts
         const clients = Object.values (this.clients || {});
         const closedClients = [];
+        for (let i = 0; i < clients.length; i++) {
+            const client = clients[i] as WsClient;
+            client.error = new ExchangeClosedByUser (this.id + ' closedByUser');
```

Comment:
@pcriadoperez is this going to trigger a reject of all the futures inside the client?

Reply:
@carlosmiei correct. I think it's safe it's the same behavior we had in the past and also the behavior followed in the other languages clients. But it was missing in typescript.